AirCasting/SessionViews/SessionHeaderView.swift

```diff
@@ -48,18 +48,54 @@ struct SessionHeaderView: View {
 
 private extension SessionHeaderView {
     var dateAndTime: some View {
```

Comment:
this variable is super big& complicated! Maybe we could extract it somewhere as a function?